dev/TreeView/InteractionTests/TreeViewTests.cs

```diff
@@ -2578,6 +2578,38 @@ public void TreeViewNodeInMarkupTest()
             }
         }
 
+        [TestMethod]
+        [TestProperty("TestSuite", "B")]
+        public void TreeViewRootNodeBindingTest()
```

Comment:
@RBrid there are some async updates when ItemsSource changes, I feel it's easier to do it in interaction test compare to API test (especially with all your helper function in place to repro the issue 😊).